### ISSUE TITLE
Bun removes whole word with a single backspace in Windows 11 with Powershell 7

### ISSUE THREAD

**Issue by deadcoder0904** [oven-sh/bun#10017]
### What version of Bun is running?

1.1.2+c8d072c2a

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

Run `bun index.ts`

### index.ts

```ts
const title = prompt("Enter title: ")
const tags = prompt("Enter tags: ")

const removeNewLines = (str: string) => str.replace("\r","")

const metadata = `---
title: '${removeNewLines(title)}'
date: '${new Date()}'
tags: ${JSON.stringify(removeNewLines(tags).replace(" ", "").split(","))}
draft: true
---
`

const dir = removeNewLines(title).toLowerCase().replace(" ", "-")
await Bun.write(`${dir}/index.md`, metadata);
```

### What is the expected behavior?

Shouldn't delete the whole thing

### What do you see instead?

Removes the whole world which makes it useless if I make typo & since I type fast, I make typos & course-correct.

### Additional information

Improve Docs on how to take CLI inputs from terminal.

Also, the input prompt looks like it has 1 extra space between `: H` below on both:

```bash
Enter title:  Hell World
Enter tags:  hell, world
```

**Comment by deadcoder0904** [comment]
@dylan-conway there is also an issue of when I use `CTRL+C` when it asks for title, it prints `Enter title:  Enter tags: ` whereas Idk if I should expect it to print `Enter title: ` only.

**Comment by deadcoder0904** [comment]
@Jarred-Sumner this isn't in v1.1.3 release, right?

**Comment by deadcoder0904** [comment]
Can confirm, this got fixed with v1.1.4 I think as I can currently do backspace letter by letter.

@dylan-conway is there a way to fix an extra space in `prompt`?

Here's my script `index.ts`:

```ts
import slugify from '@sindresorhus/slugify'

const title = prompt('Enter title: ') || 'Anon'
const tags = prompt('Enter tags: ') || '[]'

const removeNewLines = (str: string) => str.replace('\r', '')

const metadata = `---
title: '${removeNewLines(title)}'
date: '${new Date()}'
tags: ${JSON.stringify(removeNewLines(tags).trim().split(','))}
draft: true
---
`

const dir = slugify(removeNewLines(title))
await Bun.write(`./_posts/${dir}/index.md`, metadata)
```
